### PR TITLE
Fixes reflector rotation by shuttles

### LINKED ISSUE #710: [BUG] Reflectors don't rotate when a ship is launched.
## Round ID:
[629](https://shiptest.ga/stats/rounds/629)

## Testmerges:
#230 
#652 
#607 
#530 

## Reproduction:
Put a reflector on a ship, then launch the ship. Notice the reflectors don't rotate, despite the ship rotating when it launches.

### PR DESCRIPTION
**Pull request by benbot16**
## About The Pull Request

This PR fixes reflectors not rotating when a shuttle takes off/lands. 

## Why It's Good For The Game

Having to re-rotate the reflectors in an SM setup every time the ship decides to move is annoying.

closes #710 

## Changelog
:cl:
fix: Reflectors on shuttles rotate properly now.
/:cl:

### CODE REVIEW

File: code/modules/shuttle/shuttle_rotate.dm [hunk 1]
```diff
@@ -80,6 +80,10 @@ If ever any of these procs are useful for non-shuttles, rename it to proc/rotate
 	params &= ~ROTATE_OFFSET
 	return ..()
 
+/obj/structure/reflector/shuttleRotate(rotation, params)
+	. = ..()
+	setAngle(SIMPLIFY_DEGREES(rotation_angle += rotation))
```
Comment (MarkSuckerberg) [inline code/modules/shuttle/shuttle_rotate.dm]:
```suggestion
	setAngle(SIMPLIFY_DEGREES(rotation_angle + rotation))
``` 
might only need to be this? I could be wrong

Comment (benbot16) [reply code/modules/shuttle/shuttle_rotate.dm]: setAngle already sets rotation_angle, and I need SIMPLIFY_DEGREES to check for degrees greater than 360. Rotation_angle + rotation is just getting the new angle we want to set.

Comment (MarkSuckerberg) [reply code/modules/shuttle/shuttle_rotate.dm]: Yep, in that case you'll only need + since += adds and then sets the var to it.

Comment (benbot16) [reply code/modules/shuttle/shuttle_rotate.dm]: Thought those two were flipped, evidently I just can't read.

**Pull request merged**